docs/settings.rst

```diff
@@ -41,3 +41,8 @@ The name of the cache (from the ``CACHES`` dict) to use. Defaults to
 The string import path to a view to use when a request is ratelimited, in
 conjunction with ``RatelimitMiddleware``, e.g. ``'myapp.views.ratelimited'``.
 Has no default - you must set this to use ``RatelimitMiddleware``.
+
+``RATELIMIT_FAIL_OPEN``
+------------------
+
+Whether to allow requests when the cache backend fails. Defaults to ``True``.
```

Comment:
Hmm, as I think about it, the default here should probably be `False`. Allowing all requests is probably unexpected failure behavior.